react/features/presence-status/components/PresenceLabel.js

```diff
@@ -125,7 +130,8 @@ function _mapStateToProps(state, ownProps) {
 
     return {
         _presence:
-            (participant && participant.presence) || ownProps.defaultPresence
+            (participant && participant.presence) || ownProps.defaultPresence,
+        _disabled: presenceStatusDisabled()
```

Comment:
This achieves the same as setting `_presence` to null. Maybe we should do that instead?